AmazonChimeSDK/AmazonChimeSDK/internal/analytics/DefaultMeetingStatsCollector.swift

```diff
@@ -68,5 +68,6 @@ import Foundation
         maxVideoTileCount = 0
         meetingStartTimeMs = 0
         meetingStartConnectingTimeMs = 0
+        // TODO: meetingHistory should also get reset
```

Comment:
why is this a TODO. Isn't it simple to reset?

Reply:
It's not as simple as to reset, there are some async issue: the meeting history gets cleared before call end callback is triggered, customers will not be able to get the history in this callback. Need to figure out the timing.